sdv/evaluation/multi_table.py

```diff
@@ -88,7 +88,7 @@ def get_column_plot(real_data, synthetic_data, metadata, table_name, column_name
 
 
 def get_column_pair_plot(real_data, synthetic_data, metadata,
-                         table_name, column_names, plot_type=None):
+                         table_name, column_names, sample_size=None, plot_type=None):
```

Comment:
can we add this as the last parameter?

Reply:
Yes, done in [588f155](https://github.com/sdv-dev/SDV/pull/1672/commits/588f155d77a097b4601802a3a06ea567d62c2f7d)